docs/source/guides/getting_started.rst

```diff
@@ -380,7 +401,50 @@ the implementation of your service. What we are able to do is build a client who
 on the same JAX-RS annotations you use for your server. That makes it really easy to make clients
 for your Java consumers; however, it won't help you much with consumers of other languages. Instead
 of trying to explain all the concepts that are going on within the client we recommend you take a
-look at our `client documentation <https://github.com/cerner/beadledom/tree/master/client>`_.
+look at our `client documentation <https://github.com/cerner/beadledom/tree/master/client>`_..
+
+HelloWorldResource
+++++++++++++++++++
+
+.. code-block:: java
+
+  @Path("/hello")
+  public interface HelloWorldResource {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    GenericResponse<HelloWorldDto> getHelloWorld();
+  }
+
+This is the definition for the ``/hello`` resource from a client-side perspective. This interface
+allows automatically generating a client implementation via a Java proxy object.
+
+.. code-block:: java
+
+  /**
+   * JAX-RS annotation defining the base URI path for the resource.
```

Comment:
It's just the path for the resource. The base URI would be configured on the client rather than individual resources.

Reply:
Changed it here: https://github.com/cerner/beadledom/pull/133/commits/90d88965913cd8d3fcb84c756c7d2516e8aa2506